subsys/logging/Kconfig

```diff
@@ -279,6 +280,58 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FS
+	bool "Enable file system backend"
+	depends on FILE_SYSTEM
+	default n
+	help
+	  When enabled backend is using file system to output logs.
+
+if LOG_BACKEND_FS
+
+config LOG_BACKEND_FS_AUTOSTART
+	bool "Enable file system backend start automatically"
+	default y
+	help
+	  Allow backend to start autmatically, initialize it's own
```

Comment:
autmatically->automatically
shouldn't it mention that littlefs is being is used?